### ISSUE TITLE
[BUG] Text message demanding the FSR4 code.

### ISSUE THREAD

**Issue by cjdduarte** [open-spaced-repetition/fsrs4anki-helper#216]
I have two profiles on my Anki:
One official that I use for studying and is correctly configured with the FSR4 code.
Another profile that I use for testing or importing decks. In this one, I don't use FSR4. However, when I try to advance the cards, it requires the code when I try to answer the card.

![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/42243532/7bd66e48-1140-45b1-9c54-9d58a96b8946)

**Comment by L-M-Sherlock** [comment]
Do you enable the auto-disperse or auto-reschedule?

**Comment by cjdduarte** [comment]
I noticed that this one is marked by default.
![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/42243532/23746599-ba3d-4d1d-80f9-f8abd28cffd6)

When I unchecked it, it worked, but it also unchecked for my main profile.
In other words, if I need to switch between profiles, I have to keep checking or unchecking. But it solved the issue.

Would it be possible for the options to be saved per profile?
